changelog_entries/wc-double-scrollbar-bugfix.md

```diff
@@ -0,0 +1,3 @@
+ ### Campaigns
+    * World Conquest
+      Fix World Conquest's Help dialog showing double scrollbars due to the left tree having it's vertical scrollbar disabled. Changed both vertical and horizontal scrollbar modes to automatic. (issue #8576)
```

Comment:
It's a bit late to notice, but this changelog is out of date, it no longer changes the horizontal mode.